grammar.js

```diff
@@ -626,7 +633,6 @@ module.exports = grammar({
         "|.",
         "|="
       ),
-    infix: ($) => "infix",
```

Comment:
Did you mean to remove this?

Reply:
Yeah, it said it was a duplicate and it looked like it.

Reply:
Right, sorry I though I fixed that already. :)